CHANGES.md

```diff
@@ -11,7 +11,7 @@ To be released.
  -  Added the parameter `protocolVersion` to `Block<T>(long, long, BigInteger,
     Nonce, Address?, HashDigest<SHA256>?, DateTimeOffset,
     IEnumerable<Transaction<T>> transactions, HashDigest<SHA256>?,
-    HashDigest<SHA256>?)` constructor.  [[#1142], [#1147]]
+    HashDigest<SHA256>?)` constructor.  [[#1142], [#1147], [#1162]]
  -  Added the parameter to `protocolVersion` to `Block<T>.Mine()` method.
     [[#1142], [#1147]]
  -  Added the first parameter `protocolVersion` to `BlockHeader()` constructor.
```

Comment:
I tried to pull request links on the relevant changelogs. Should these lines be updated also? I didn't because I thought this pull request is not related to the constructors, but if I see it as the feature, `protocol version`, they are related. How do you think @dahlia?

Reply:
It seems relevant too.

Reply:
I added also them, thanks 😀